packages/slider/index.d.ts

```diff
@@ -0,0 +1,37 @@
+declare module "@reach/slider" {
+  export type SliderProps = React.HTMLProps<HTMLDivElement> & {
```

Comment:
I think you might need a `import * as React from 'react';` so you can use `React` types? Not 100% sure.